### PR TITLE
ARUHA-188 Send status code for streaming right after allocating resou…

### PR DESCRIPTION
**Pull request by antban**
…rces to handle client

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/233" target="_blank">ReviewNinja

File: src/main/java/de/zalando/aruha/nakadi/service/EventStream.java [hunk 1]
```diff
@@ -42,15 +40,16 @@ public EventStream(final EventConsumer eventConsumer, final OutputStream outputS
 
     public void streamEvents() {
         try {
+            outputStream.flush();
```
Comment (v-stepanov) [inline src/main/java/de/zalando/aruha/nakadi/service/EventStream.java]: I thought you will do that in controller, seems to be a more suitable place from logic separation point of view.
What do you think?

Comment (antban) [inline src/main/java/de/zalando/aruha/nakadi/service/EventStream.java]: My mistake, I thought that I was in controller :(

File: src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java [hunk 2]
```diff
@@ -9,41 +9,39 @@
...
-import static de.zalando.aruha.nakadi.metrics.MetricUtils.metricNameFor;
+import javax.annotation.Nullable;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
 import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
 import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
 import static javax.ws.rs.core.Response.Status.NOT_FOUND;
 import static javax.ws.rs.core.Response.Status.PRECONDITION_FAILED;
 import static javax.ws.rs.core.Response.Status.SERVICE_UNAVAILABLE;
+import org.slf4j.Logger;
```
Comment (v-stepanov) [inline src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java]: Every time you reorder the imports a kitten dies

**Comment by v-stepanov** [comment]
:shipit:

File: src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java (hunk 2 above)
Comment (antban) [inline src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java]: Oh man, didn't so that, will revert reordering.

Comment (v-stepanov) [inline src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java]: It's ok, no need for that :)

Comment (antban) [inline src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java]: I don't want to kill kittens. Kitten resurrected

**Pull request merged**